addons/web_editor/views/snippets.xml

```diff
@@ -731,6 +731,48 @@
                 Reset
             </we-button>
         </we-row>
+        <div id="cropTools">
+            <div>
+                <we-row string="└ Aspect Ratio">
+                        <we-button class="cropBtn" data-action="ratio" data-no-preview="true" data-value="0/0">Flexible</we-button>
+                </we-row>
+                <we-row string=" ">
+                        <we-button class="cropBtn" data-action="ratio" data-no-preview="true" data-value="16/9">16:9</we-button>
+                        <we-button class="cropBtn" data-action="ratio" data-no-preview="true" data-value="4/3" >4:3</we-button>
+                        <we-button class="cropBtn" data-action="ratio" data-no-preview="true" data-value="1/1" >1:1</we-button>
+                        <we-button class="cropBtn" data-action="ratio" data-no-preview="true" data-value="2/3" >2:3</we-button>
+                </we-row>
+            </div>
+            <div>
+                <we-row string="└ Zoom">
```

Comment:
@maad-odoo adding class "o_we_sublevel_1" will add  '└' for you. A good catch by @deso-odoo :wink: